cmd/thanos/main.go

```diff
@@ -78,7 +78,7 @@ func main() {
 
 	metrics := prometheus.NewRegistry()
 	metrics.MustRegister(
-		version.NewCollector("prometheus"),
+		version.NewCollector(fmt.Sprintf("thanos-%s", cmd)),
```

Comment:
Not just "thanos"? It's one release artifact in the end, right?

Reply:
yes, but imagine having lot's of different sidecars, queries in the system. Yeah I think we will be able to distinguish them by some instance label or something (: